src/connectors/infinite-hits/connectInfiniteHits.js

```diff
@@ -70,23 +70,39 @@ export default function connectInfiniteHits(renderFn, unmountFn) {
   return (widgetParams = {}) => {
     const { escapeHTML = true, transformItems = items => items } = widgetParams;
     let hitsCache = [];
+    let firstReceivedPage = Infinity;
     let lastReceivedPage = -1;
 
-    const getShowMore = helper => () => helper.nextPage().search();
+    const getShowPrevious = helper => () => {
+      helper
+        .overrideStateWithoutTriggeringChangeEvent({
```

Comment:
Is there a reason for not using `setPage` as below? If so, can we add a comment explaining why?

Reply:
Yes, here we're using `overrideStateWithoutTriggeringChangeEvent` to update the internal state without updating the UI since we do not want to change the page number in the URL ([RFC](https://github.com/algolia/instantsearch-rfcs/blob/master/accepted/infinite-hits-previous-page.md#detailed-design)).

I will add a comment to make it clearer.

Reply:
I see! This is not caught by any tests though. We should add one.

Reply:
Added the following assertion in `connectInfiniteHits-test.js` test suite:

```js
expect(helper.emit).not.toHaveBeenCalled();
```